js/settings.js

```diff
@@ -36,7 +37,8 @@ Settings.SETTINGS = {
   'spacestab': {'default': true, 'type': 'boolean', 'widget': 'checkbox'},
   'tabsize': {'default': 4, 'type': 'integer', 'widget': 'number'},
   'theme': {'default': 'default', 'type': 'string', 'widget': 'radio'},
-  'wraplines': {'default': true, 'type': 'boolean', 'widget': 'checkbox'}
+  'wraplines': {'default': true, 'type': 'boolean', 'widget': 'checkbox'},
+  'search': {'default': true, 'type': 'boolean', 'widget': 'search'}
```

Comment:
nit: add trailing comma to the end of a js obj as per google style guide

Reply:
thanks!